### DISCUSSION TITLE
Can the random number of element to be picked in faker.helper.arrayElements() be zero?

### DISCUSSION THREAD

**Discussion by hazem-alabiad** [faker-js/faker#1002] [Q&A] [answered]
If I left the count parameter in `faker.helper.arrayElements()` unset, does the number of elements to be picked be randomly zero?

**Comment by xDivisionByZerox** [comment, marked as answer]
This is possible in some edge cases:
1. You provide a `count` parameter of `0` 
2. You provide a `count` parameter of less than `0` (the value will get defaulted to 0) https://github.com/faker-js/faker/blob/9432eddae7ce2451ad68d5fccda426df059073c5/src/modules/helpers/index.ts#L455-L457
3. You provide an `array` parameter an empty array https://github.com/faker-js/faker/blob/9432eddae7ce2451ad68d5fccda426df059073c5/src/modules/helpers/index.ts#L453-L455

So in the case, you stated this will not happen, since the count parameter will default to a random value between 1 and the array length https://github.com/faker-js/faker/blob/9432eddae7ce2451ad68d5fccda426df059073c5/src/modules/helpers/index.ts#L452
